sound/soc/bcm/rpi-wm8804-soundcard.c

```diff
@@ -372,6 +512,12 @@ static int snd_rpi_wm8804_probe(struct platform_device *pdev)
 		snd_clk48gpio =
 			devm_gpiod_get(&pdev->dev, "clock48", GPIOD_OUT_LOW);
 
+		if(of_property_read_u32(pdev->dev.of_node, "sys_clk", &sysclk_freq))
```

Comment:
I meant all of it, including the sys_clk logic - you have no reason to make changes for the other drivers, particularly the kernel log spam.

Reply:
okay sounds good changes have been made